### ISSUE TITLE
Issue updating to OWLAPI 4.5.4

### ISSUE THREAD

**Issue by rctauber** [ontodev/robot#346]
When updating to OWLAPI 4.5.4 (for #308), the following command in `mvn verify` fails:
```
robot relax  --input ribosome.owl --output results/relaxed.owl
```

`results/relaxed.owl` loads with a new class: `Error1` which replaces the intended relaxed axiom `'part_of' some organelle`. Upon looking at the XML for `results/relaxed.owl`, this is changed:
```
<rdfs:subClassOf rdf:parseType="Collection">
  <owl:Restriction>
    <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000050"/>
    <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/GO_0043226"/>
  </owl:Restriction>
</rdfs:subClassOf>
```

The `subClassOf` tag should not include `rdf:parseType="Collection"`, which appears to be copied from the equivalent class intersection.

I'm not sure if this is an issue with `RelaxOperation`, or if this is a bug in OWLAPI.

**Comment by cmungall** [comment]
I would assume as you do that the test data is configured incorrectly.

**Comment by rctauber** [comment]
There are other `rdfs:subClassOf` tags that have been created via `relax` that do not include the `rdf:parseType`. It's only on the *first* added one. Their equivalent class statements are all similar in structure.

So, I removed `GO:0000313` from the input file. And now, the "error" moved to `GO:0044422` in the output file. I converted the input file (`ribosome.owl`) to RDF/XML format, and it works properly, so it's only an error in the Manchester syntax version. I converted the `uberon_module.owl` test file to Manchester and ran relax on it as well, which was successful.

So it does seem to be an issue with the input file, but I can't for the life of me see anything wrong. I even tried re-creating it in Protege and it still popped up with the same issue. I haven't seen this issue with any other files I've tried `relax` on, but I'd still like to know what's causing this...
